fastlane_bot/tests/test_072_TestTerraformer.py

```diff
@@ -0,0 +1,123 @@
+
+'''
+This module tests the Terraformer
+'''
+import pytest
+import asyncio
+
+from unittest.mock import MagicMock, AsyncMock
+
+from json import dumps
+from pytest import fixture
+from dataclasses import dataclass
+from run_blockchain_terraformer import organize_pool_details_balancer, organize_pool_details_uni_v2, organize_pool_details_solidly_v2, organize_pool_details_uni_v3
```

Comment:
The following modules aren't being used; please remove the corresponding `import` statements:
```
import pytest
import asyncio
from json import dumps
from pytest import fixture
from dataclasses import dataclass
```

Reply:
Done - thank you.